lib/quantity.ex

```diff
@@ -184,6 +223,34 @@ defmodule Quantity do
   @spec to_zero(t) :: t
   def to_zero(%{unit: unit, value: %Decimal{exp: exp}}), do: Quantity.new(0, exp, unit)
 
+  @doc """
+  Converts the quantity to have a new unit.
+  The new unit must be a whole 10-exponent more or less than the original unit.
+  The exponent given is how much the new unit is more than the old unit
+
+  iex> ~Q[1234E3 Wh] |> Quantity.convert_unit("MWh", 6)
+  ~Q[1.234 MWh]
```

Comment:
:+1: Nice function!

I would suggest another wording in L229, perhaps "The exponent given is difference in exponents as exponent_new - exponent_old. For example 6 - 0 = 6 when converting to MWh from Wh"

Also, I would suggest one more doctest for an operation in the other direction (e.g. from meters to millimeters).

Reply:
I agree. Thanks for the good suggestions.
Changed and force-pushed. Changes can be seen here: https://github.com/wise-home/quantity/blob/299f29dc595cce6e52802219acd1a474320d0d79/lib/quantity.ex#L226-L238

Reply:
Very nice :+1: